src/ImageRunModal.jsx

```diff
@@ -824,222 +824,224 @@ export class ImageRunModal extends React.Component {
-                              onPlus={() => this.onPlusOne('restartTries')}
-                              onChange={ev => this.onNumberValue('restartTries', ev.target.value, 1)}
-                                    />
+
+                            {this.isSystem() &&
+                                <FormGroup
+                                    fieldId='run-image-cpu-priority'
+                                    label={_("CPU shares")}
+                                    labelHelp={
+                                        <Popover aria-label={_("CPU Shares help")}
+                                            enableFlip
+                                            bodyContent={_("CPU shares determine the priority of running containers. Default priority is 1024. A higher number prioritizes this container. A lower number decreases priority.")}>
+                                            <Button variant="plain" hasNoPadding aria-label="More info" icon={<OutlinedQuestionCircleIcon />} />
+                                        </Popover>
+                                    }>
+                                    <Flex alignItems={{ default: 'alignItemsCenter' }} className="ct-input-group-spacer-sm modal-run-limiter" id="run-image-dialog-cpu-priority">
+                                        <Checkbox id="run-image-dialog-cpu-priority-checkbox"
+                                            isChecked={this.state.cpuSharesConfigure}
+                                            onChange={(_event, checked) => this.onValueChanged('cpuSharesConfigure', checked)} />
+                                        <NumberInput
+                                            id="run-image-cpu-priority"
+                                            value={dialogValues.cpuShares}
+                                            onClick={() => !this.state.cpuSharesConfigure && this.onValueChanged('cpuSharesConfigure', true)}
+                                            min={2}
+                                            max={262144}
+                                            isDisabled={!this.state.cpuSharesConfigure}
+                                            onPlus={() => this.onPlusOne('cpuShares')}
+                                            onMinus={() => this.onMinusOne('cpuShares')}
+                                            minusBtnAriaLabel={_("Decrease CPU shares")}
+                                            plusBtnAriaLabel={_("Increase CPU shares")}
+                                            onChange={ev => this.onNumberValue('cpuShares', ev.target.value, 2)} />
+                                    </Flex>
                                 </FormGroup>
                             }
-                        </Grid>
-                        }
+                            {((userLingeringEnabled && userPodmanRestartAvailable) || (this.isSystem() && podmanRestartAvailable)) &&
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2246-4debadf1-20250829-124849-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)